src/itwinai/loggers.py

```diff
@@ -477,14 +521,14 @@ def __init__(
         super().__init__(
             savedir=savedir, log_freq=log_freq, log_on_workers=log_on_workers
         )
-        self.experiment_name = experiment_name
         self.tracking_uri = tracking_uri
         self.run_description = run_description
         self.run_name = run_name
+        self.experiment_name = experiment_name
 
         self.tracking_uri = (
             self.tracking_uri
-            or os.environ.get('MLFLOW_TRACKING_URI')
+            or os.environ.get("MLFLOW_TRACKING_URI")
             or pathlib.Path(os.path.abspath(self.savedir)).as_uri()
```

Comment:
I know this isn't code you changed but this hurts a bit because one can just do `.resolve()` instead of `os.path.abspath`